tests/topotests/ospf6-topo1/test_ospf6_topo1.py

```diff
@@ -138,102 +148,103 @@ def build(self, **_opts):
 ##
 #####################################################
 
-def setup_module(module):
-    global topo, net
-
-    print("\n\n** %s: Setup Topology" % module.__name__)
-    print("******************************************\n")
+def setup_module(mod):
+    "Sets up the pytest environment"
 
-    print("Cleanup old Mininet runs")
-    os.system('sudo mn -c > /dev/null 2>&1')
+    tgen = Topogen(NetworkTopo, mod.__name__)
+    tgen.start_topology()
 
-    thisDir = os.path.dirname(os.path.realpath(__file__))
-    topo = NetworkTopo()
-
-    net = Mininet(controller=None, topo=topo)
-    net.start()
+    print("\n\n** %s: Setup Topology" % mod.__name__)
+    print("******************************************\n")
 
-    # For debugging after starting net, but before starting FRR/Quagga, uncomment the next line
-    # CLI(net)
+    # For debugging after starting net, but before starting FRR,
+    # uncomment the next line
+    # tgen.mininet_cli()
 
     ospf_config = 'ospf6d.conf'
-    if net['r1'].checkRouterVersion('<', '4.0'):
+    if tgen.gears['r1'].has_version('<', '4.0'):
```

Comment:
It is nice that you were able to translate this, but I think we can remove it since I think we might not want to support old versions anymore (that is one of the reasons why we've imported topotest to frr: versioned tests are attached to FRR version).

Reply:
Yes, quite - I've removed that test

Reply:
I think there are some old configuration files (the `ospf6d.conf-pre-v4` files) that should be removed with it.